### ISSUE TITLE
Todo Overlay does not appear on Ubuntu.

### ISSUE THREAD

**Issue by matthiasBender** [ahonn/logseq-plugin-todo#67]
Trying to open the Todolist using the button in the top menu in logseq results in the following stacktrace:
```
index.af8ba924.js:150 Uncaught (in promise) Error: Not existed method #queryElementRect
    at br.R (lsplugin.core.js:2:49378)
    at br.<anonymous> (lsplugin.core.js:2:93271)
    at a.emit (lsplugin.core.js:2:28521)
    at X.<anonymous> (lsplugin.core.js:2:88623)
    at lsplugin.core.js:2:53506
    at Array.forEach (<anonymous>)
    at X.listener (lsplugin.core.js:2:53491)
$h @ index.af8ba924.js:150
await in $h (async)
(anonymous) @ index.af8ba924.js:12
postMessage (async)
call @ lsplugin.core.js:2
_callUserModel @ lsplugin.core.js:2
callUserModel @ lsplugin.core.js:2
(anonymous) @ lsplugin.core.js:2
g @ helpers.js:151
```
I am using the current AppImage for Linux: `Logseq-linux-x64-0.10.2.AppImage` on an Ubuntu 20.04 installation.
The version of the Todo list is v1.20.0.

**Comment by WilliamDeBruin** [comment]
Same issue on Mac v0.10.3

**Comment by ahonn** [comment]
I apologize for the late response. 
This issue has been fixed in v1.20.1, please update plugin to the latest version.
